vm-wasmi/Cargo.toml

```diff
@@ -1,13 +1,8 @@
 [package]
 name = "cosmwasm-vm-wasmi"
-version = "0.1.0"
+version = "0.2.0"
 edition = "2021"
-authors = [
-  "Hussein Ait Lahcen hussein.aitlahcen@gmail.com",
-  "Abdullah Eryuzlu abdullaheryuzlu@gmail.com",
-  "Composable Developers",
-]
-homepage = "https://composable.finance"
```

Comment:
interesting

Reply:
feel free to join back:) I just dislike noise in headers. I will not add myself sure. Feel free to PR to add these back to the end of toml file

Reply:
or add to workspace toml, and there is even tooling to add that during release

Reply:
no i think useful to point to homepage at all - if somebody operates on source code of vm level - he either wants code, or spec. sure not marketing materials.

Reply:
so who cares if it obsolete